extensions/amp-story/1.0/amp-story.js

```diff
@@ -1632,6 +1635,9 @@ export class AmpStory extends AMP.BaseElement {
         actions.execute(this.sidebar_, 'open', /* args */ null,
             /* source */ null, /* caller */ null, /* event */ null,
             ActionTrust.HIGH);
+      } else if (this.sidebar_ && !sidebarState &&
+                    !this.maskElement_.hasAttribute('hidden')) {
+        this.closeOpacityMask_();
       } else {
         this.sidebarObserver_.disconnect();
```

Comment:
This is the code that's executed when the sidebar is closed. You can call `this.closeOpacityMask_()` from here, and I think that's all the code you need for this PR :)

Reply:
Thank you, Gabriel!